stdlib/LinearAlgebra/src/qr.jl

```diff
@@ -859,6 +865,20 @@ function ldiv!(A::QR{T}, B::StridedMatrix{T}) where T
     end
     return B
 end
+
+
+function ldiv!(A::QR{T}, B::StridedMatrix{T}) where T
+    m, n = size(A)
+    m < n && return _wide_qr_ldiv!(A, B)
+
+    mB, nB = size(B)
+    lmul!(adjoint(A.Q), view(B, 1:m, :))
+    R = A.factors
+    ldiv!(UpperTriangular(view(R,1:n,:)), view(B, 1:n, :))
+    return B
+end
+ldiv!(A::QR, B::StridedVector) = ldiv!(A, reshape(B, length(B), 1))[:]
```

Comment:
But this one allocates, doesn't it? In case that's true, for convenience:
```suggestion
function ldiv!(A::QR, B::StridedVector)
    ldiv!(A, reshape(B, length(B), 1))
    return B
end
```